testing/test_terminal.py

```diff
@@ -2079,19 +2078,19 @@ def check(msg, width, expected):
     check("some\nmessage", 80, "FAILED some::nodeid - some")
 
     # Test unicode safety.
-    check("😄😄😄😄😄\n2nd line", 25, "FAILED some::nodeid - ...")
-    check("😄😄😄😄😄\n2nd line", 26, "FAILED some::nodeid - ...")
-    check("😄😄😄😄😄\n2nd line", 27, "FAILED some::nodeid - 😄...")
-    check("😄😄😄😄😄\n2nd line", 28, "FAILED some::nodeid - 😄...")
-    check("😄😄😄😄😄\n2nd line", 29, "FAILED some::nodeid - 😄😄...")
+    check("🧛🧛🧛🧛🧛\n2nd line", 25, "FAILED some::nodeid - ...")
```

Comment:
This change is because the width of the 😄 character (U+1F604) happened to change from N (width 1) to W (width 2) in Unicode 9, but Python 3.5 uses Unicode 8, so the test started to fail there. I changed to a different character which was always wide.

Reply:
OK, Vampire doesn't work on Pypy (Unicode 9), it was only added in Unicode 10. So I changed to 🉐 now (Japanese “Bargain” Button) which should hopefully work in all versions..

Reply:
Yes, wcwidth wants to match modern terminal versions in behavior by default, to be most up-to-date. we are also trying to tackle multi-code point emoji and other difficulties, anyway best wishes from wcwidth package author.